crypto/fipsmodule/rand/internal.h

```diff
@@ -133,6 +133,61 @@ OPENSSL_EXPORT void HAZMAT_set_urandom_test_mode_for_testing(void);
 #define JITTER_MAX_NUM_TRIES (3)
 #endif
 
+
+#if defined(AWSLC_FIPS)
+
+// CRNGT_BLOCK_SIZE is the number of bytes in a “block” for the purposes of the
+// continuous random number generator test in FIPS 140-2, section 4.9.2.
+#define CRNGT_BLOCK_SIZE 16
+
+#define FIPS_CRNGT_TEST_SIZE CRNGT_BLOCK_SIZE
+#define AWSLC_PASSIVE_ENTROPY_OVERREAD_MULTIPLER 10
+#define AWSLC_PASSIVE_ENTROPY_SOURCE_DRAW_SIZE (FIPS_CRNGT_TEST_SIZE + (CTR_DRBG_ENTROPY_LEN * AWSLC_PASSIVE_ENTROPY_OVERREAD_MULTIPLER))
+#define ENTROPY_POOL_SIZE AWSLC_PASSIVE_ENTROPY_SOURCE_DRAW_SIZE
+
+// Fixed-sized flat memory buffer definition used for passive entropy
+// implementation.
+struct entropy_pool {
+  size_t capacity;
```

Comment:
I went back and forward on this. I think it makes the buffer self-contained (i.e. encodes the size of the data it can maintain). And more readable IMO.

Also allows it being a bit more dynamic later on, perhaps, if needed.